storage/badger/operation/prefix.go

```diff
@@ -45,6 +45,7 @@ const (
 	codeExecutionReceiptMeta = 36
 	codeResultApproval       = 37
 	codeChunk                = 38
+	codeQuorumCertificate    = 39
```

Comment:
conceptually, these codes are all for storing entities by their respective ID. For QCs, this would be storing `qc` by `qc.ID()`. But that is not an index we implement. 

We index the `qc` by `qc.BlockID`. This is very similar to [indexing a `seal` by `seal.BlockID`](https://github.com/onflow/flow-go/blob/fc9cf312e52f4b32d47d5169e7f0e60d8c4fab9f/storage/badger/operation/prefix.go#L56). The codes for such indices are defined in the group below:
https://github.com/onflow/flow-go/blob/fc9cf312e52f4b32d47d5169e7f0e60d8c4fab9f/storage/badger/operation/prefix.go#L50

Hence, I would assign the code `45`
```golang
codeBlockIDToQuorumCertificate = 45 // index QuorumCertificate by the block ID it certifies
```